### ISSUE TITLE
Electron not updating stats before restart

### ISSUE THREAD

**Issue by cryptobench** [golemfactory/golem-electron#568]
Creating issue on behalf of Robert.stohler + cryptomike from the rocket chat. 
https://chat.golem.network/channel/chit-chat?msg=zsiKEJJeD62L6JR4e
https://chat.golem.network/channel/chit-chat?msg=bgcxbQmj5ewJx4H7z

User kyteli54 is experiencing this issue on terminal also, so i'm not sure if it's an electron issue after all.

https://chat.golem.network/channel/chit-chat?msg=dW6mk5dyiwEebcrik

**I've asked for OS and logs etc and will update the issue once I get them.**
## Description
New stats in the bottom of the gui and transactions are not shown in the GUI before a restart. 

**Electron version**:
0.20.1 Golem Release

**Golem Version**:
0.20.1

**Golem-Messages version** (leave empty if unsure):

**OS** [e.g. Windows 10 Pro]:
@cryptomike - Windows 10
**Branch** (if launched from source):

**Mainnet/Testnet**:
Mainnet
**Description of the issue**:
The stats for successful stats and errors etc are not updating when completed. They are not shown in the GUI before a restart of the GUI.

_A clear and concise description of what went wrong, in which component, when and where._

**Actual result**:

_What is the observed behavior and/or result in this issue_

**Screenshots**:

_Add screenshots to help explain your problem._

## Steps To Reproduce
1. Launch Golem
2. Wait and see nothing has happened in the GUI
3. restart golem
4. see stats are now updated.

## Expected behavior
Stats are updating live.

## Logs and any additional context
@cryptomike logs - 
[golem-log-cryptomike.zip](https://github.com/golemfactory/golem-electron/files/3423951/golem-logs.1.zip)

## Proposed Solution?
_(Optional: What could be a solution for that issue)_

**Comment by mdtanrikulu** [comment]
Hi! Thank you for the issue with all details. It was quite helpful. There was a problem in deep equal check function, it should be resolved for the next release.
